EIPS/eip-eof-tracing.md

```diff
@@ -0,0 +1,173 @@
+---
+eip: TBD
+title: EOF/EVM Trace Specification
+author: Martin Holst Swende (@holiman), Marius van der Wijden (@MariusVanDerWijden), Danno Ferrin (@shemnon)
+discussions-to: TBD
```

Comment:
Please create a topic in Eth Magicians.  It just needs to contain a link to this PR.